src/site/content/en/progressive-web-apps/define-install-strategy/index.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: How to define your install strategy
+authors:
+  - demianrenzulli
+  - petelepage
+date: 2020-05-12
+updated: 2020-05-13
+description: |
+  Best practices for combining different installation offerings to increase installation rates and avoid platform competition and cannibalization.
+---
+
+In the past, app installs were only possible in the context of native applications. Today, modern web apps offer installable experiences that provide the same level of integration and reliability as native apps.
+
+You can achieve this in different ways:
+
+- Installing the PWA [from the browser](https://web.dev/customize-install/).
+- Installing the PWA [from the app store](https://developers.google.com/web/android/trusted-web-activity).
+
+Having different distribution channels is a powerful way of reaching a broad number of users, but choosing the right strategy to promote them, can be challenging.
+
+This guide explores best practices for combining different installation offerings to increase installation rates and avoid platform competition and cannibalization. The installation offerings covered include PWAs installed from both the browser and the App Store, as well as native apps.
+
+## Why make your web app installable?
+
+Installed Progressive Web Apps run in a standalone window instead of a browser tab. They're launchable from on the user's home screen, dock, taskbar, or shelf. It's possible to search for them on a device and jump between them with the app switcher, making them feel like part of the device they're installed on. 
```

Comment:
```suggestion
Installed Progressive Web Apps run in a standalone window instead of a browser tab. They're launchable from the user's home screen, dock, taskbar, or shelf. It's possible to search for them on a device and jump between them with the app switcher, making them feel like part of the device they're installed on. 
```